src/core/route/component/not-found/NotFoundPage.tsx

```diff
@@ -1,8 +1,7 @@
 import React, {FunctionComponent, useState} from "react";
-import {RouteComponentProps} from "@reach/router";
-import ReportModal from "./modal/components/report/ReportModal";
+import ReportModal from "../../../../components/modal/components/report/ReportModal";
 
-const NotFound: FunctionComponent<RouteComponentProps> = () => {
+const NotFoundPage: FunctionComponent = () => {
```

Comment:
Let's make sure all components in the template follow our practices

```suggestion
function NotFoundPage() {
```